src/components/BitcoinLineChart/index.js

```diff
@@ -0,0 +1,108 @@
+import axios from 'axios'
+import React from 'react'
+import { Line } from 'react-chartjs-2'
+
+const options = {
+  plugins: {
+    legend: {
+      display: false,
+    },
+  },
+  elements: {
+    point: {
+      radius: 0,
+    },
+    line: {
+      tension: 0.4,
+    },
+  },
+  scales: {
+    y: {
+      grid: {
+        display: false,
+        drawTicks: false,
+        borderWidth: 0,
+      },
+      ticks: {
+        display: false,
+      },
+    },
+    x: {
+      grid: {
+        display: false,
+        borderWidth: 0,
+      },
+      ticks: {
+        autoSkip: true,
+        maxTicksLimit: 14,
+        maxRotation: 0,
+        minRotation: 0,
+      },
+    },
+  },
+}
+
+class BitcoinLineChart extends React.Component {
+  state = {
+    dataPoints: [],
+    dataLabels: [],
+    gradient: '',
+  }
+
+  getPrices = async () => {
+    try {
+      const { data } = await axios(
+        `https://api.coingecko.com/api/v3/coins/bitcoin/market_chart?vs_currency=${this.props.currency}&days=39&interval=daily`,
+      )
+
+      const prices = data.prices
+      const dataPoints = []
+      const dataLabels = []
+      prices.forEach((price) => {
+        let myDay = new Date(price[0]).getDate().toString()
+        if (myDay.length === 1) {
+          myDay = '0' + myDay
+        }
+        dataLabels.push(myDay)
+        dataPoints.push(price[1])
+      })

```

Comment:
same here

Reply:
I fixed this on the pull request 2 as it's not part of this feature.